builtin/providers/aws/resource_aws_api_gateway_domain_name.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsApiGatewayDomainName() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate_upload_date": &schema.Schema{
```

Comment:
We declare this parameter in the schema but never actually set the value to state...